game/renderware_binary_stream.ksy

```diff
@@ -23,6 +23,9 @@ seq:
         sections::geometry_list: list_with_header
         sections::texture_dictionary: list_with_header
         sections::texture_native: list_with_header
+instances:
+    version:
+      value: (library_id_stamp >> 16) + 0x30000
```

Comment:
Please fix this formatting to use 2 spaces.

Reply:
Done, missed that when pasting in the changes.